libs/books/feature/src/lib/book-search/book-grid/book-grid.component.html

```diff
@@ -0,0 +1,10 @@
+<ng-container *ngIf="books?.length">
+  <tmo-book-grid-item data-testing="book-item" *ngFor="let b of books" [book]=b></tmo-book-grid-item>
+</ng-container>
+<ng-template #empty>
```

Comment:
Will this ever be shown?

Reply:
no, ng-container above is missing `else empty`